src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -353,7 +352,7 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
             }
             catch (Exception ex)
             {
-                Console.Error.WriteLine($"[ERROR] {ex.ToString()}");
+                Console.Error.WriteLine($"[ERROR] {ex.Message}");
```

Comment:
Should I add an additional catch clause for the InvalidOperationException? I assume not since this exception isn't specific enough to indicate that a process could not be started correctly.